lib/config/index.ts

```diff
@@ -14,7 +14,7 @@ import { environment } from './environment'
 import { dockerSecret } from './dockerSecret'
 import deepFreeze = require('deep-freeze')
 
-const appRootPath = path.resolve(__dirname, '../../')
+const appRootPath = process.cwd()
```

Comment:
Please don't. This breaks every CodiMD installation that doesn't explicitly specifies a work directory. (Which means probably all installations running on Systemd with minimal service files)

The app root directory is intentionally not the work directory.

Reply:
The problem I tried to solve with that change was that we can't reliably get the app root path relative to the current file anymore. If we are running in a node process that understands TypeScript directly (`ts-node` typically) we would have to go "2 directories up". When the complied code is running (so a normal NodeJS after running `tsc`) we would have to go "3 directories up", because the complied code resides in the `built` subdirectory.

So either CodiMS would not work in one of those cases or we have to build some kind of heuristic ("if there is a `built` directory, go 3 dirs up, else 2"). Personally, I would prefer to use the working directory so we don't have to do that, but I agree that we probably should discuss this.

Proposal: Use `process.cwd()` for now, as it works in both use-cases and discuss this topic in a separate issue before release.

Reply:
In order to move this forward: How about we use the current working directory for now and add a TODO hint, to fix this in a better way in the long term? Then this shouldn't block your progress.

Reply:
I'll resolve this, as we found a solution in #360. As soon as #365 is merged I'll rebase this PR and we can merge :tada: